src/app/components/rubric-action-button-group/rubric-action-button-group.component.html

```diff
@@ -1,28 +1,33 @@
-<div class="btn-group d-print-none" [class.float-md-right]="floatRight" role="group" aria-label="Action buttons">
-  <button (click)="copyLink()" type="button" class="btn btn-secondary">
-    <span *ngIf="!copied" [ngbTooltip]="'Copy Link'" class="oi oi-link-intact"></span>
-    <span *ngIf="copied" [ngbTooltip]="'Link Copied!'" class="oi oi-check"></span>
-  </button>
-  <button *ngIf="rubric.uuid"
-          type="button"
-          class="btn btn-secondary"
-          [routerLink]="'/rubrics/' + rubric.uuid + '/duplicate'">
-    <span [ngbTooltip]="'Duplicate Rubric'" class="oi oi-layers"></span>
-  </button>
-  <button *ngIf="showPrint" type="button" class="btn btn-secondary" (click)="printRubric()">
-    <span [ngbTooltip]="'Print/Download Rubric'" class="oi oi-print"></span>
-  </button>
-  <button *ngIf="rubric.private" type="button" class="btn btn-secondary" (click)="openDeleteRubricModal()">
-    <span [ngbTooltip]="'Delete Rubric'" class="oi oi-trash"></span>
-  </button>
-  <button *ngIf="rubric.created_at" [class.extra-wide]="!rubric.private"
-          type="button" class="d-none d-lg-block btn btn-outline-secondary disabled btn-date">
-    {{ rubric.created_at | date: 'mediumDate' }}
-  </button>
-  <button *ngIf="!rubric.private" type="button" class="btn btn-secondary disabled">
-    <span [ngbTooltip]="'Public'" class="oi oi-lock-unlocked"></span>
-  </button>
-  <button *ngIf="rubric.private" type="button" class="btn btn-secondary disabled">
-    <span [ngbTooltip]="'Private'" class="oi oi-lock-locked"></span>
-  </button>
-</div>
+<div class="btn-group d-print-none" [class.float-md-right]="floatRight" role="group">
+    <div aria-label="Action buttons">
+      <button (click)="copyLink()" type="button" class="btn btn-secondary">
+        <span *ngIf="!copied" [ngbTooltip]="'Copy Link'" class="oi oi-link-intact" aria-label="Copy Link"></span>
+        <span *ngIf="copied" [ngbTooltip]="'Link Copied!'" class="oi oi-check" aria-label="Link Copied"></span>
```

Comment:
I had not thought about tooltips and screen readers either. I'm guessing the labels here make it more clear what the action taken was?